style-dictionary/visual-refresh/contexts/flashbar-warning.ts

```diff
@@ -20,6 +20,10 @@ export const sharedTokens: StyleDictionary.ColorsDictionary = {
   colorTextInteractiveInvertedDefault: '{colorGrey600}',
   colorTextInteractiveInvertedHover: '{colorGrey900}',
 
+  // Progress bar should be using variant="flash", but this makes them use a white background.
```

Comment:
I find this comment a bit confusing: isn't the progress bar now _not_ using a white background exactly because of these lines? Both `{colorGrey900}` and `{colorGreyOpaque10}` are quite dark. To me it looks like the comment describes the old state that no longer applies after this PR